tools/clang/test/CodeGenDXIL/reflection/type_sys_dump_mat_as_array_of_vector.ll

```diff
@@ -0,0 +1,114 @@
+; RUN: %dxa %s -o %t
+; RUN: %dxa %t -dumpreflection | FileCheck %s
+
+target datalayout = "e-m:e-p:32:32-i1:32-i8:32-i16:32-i32:32-i64:64-f16:32-f32:32-f64:64-n8:16:32:64"
+target triple = "dxil-ms-dx"
+
+;struct S2 {
+;    row_major int4x3 m;
+;	int1x3 m1[2];
+;	int3x1 m2;
+;};
+
+; Make sure cbuffer reflection is correct with High level struct stripped: [rows x <cols x float>] and High level struct stripped, one row: <cols x float>
```

Comment:
I don't think this part of the comment is right: `one row: <cols x float>`.  This applies to the _2 variant too.

We will definitely test the single-vector case, so if we have a chance to test a `[3 x <1 x i32>]` or `[1 x <3 x i32>]`, we shouldn't squander it, right?